thirdeye/thirdeye-frontend/app/helpers/utils.js

```diff
@@ -155,7 +155,7 @@ export function appendTail(urn, tail) {
 }
 
 export function appendFilters(urn, filters) {
-  const tail = filters.map(t => `${t[0]}=${t[1]}`);
+  const tail = filters.map(t => encodeURIComponent(`${t[0]}=${t[1]}`));
```

Comment:
nit: Let's add some JS doc for those utils functions when we get a chance

Reply:
yes, i'll push a separate PR for that